src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -1,11 +1,11 @@
+import { AuthIntent, ContextModule } from "@artsy/cohesion/dist/Schema"
```

Comment:
Left a comment in cohesion, but that should allow us to not need to drill into subdirectories of the package.